ganache-with-protocol/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM node:10-alpine as builder-node
+USER root
+WORKDIR /
+
+ENV GANACHE_DB_PATH /ganache_db
+
+RUN apk add --no-cache git python bash make g++
+
+COPY build-ganache.sh build-ganache.sh
+RUN chmod +x build-ganache.sh
+
+# RUN GANACHE
+RUN ./build-ganache.sh
+
+FROM node:10-alpine
+ENV GANACHE_DB_PATH /ganache_db
+
+RUN apk add --no-cache git python bash make g++
+RUN npm i ganache-cli -g
+
+COPY --from=builder-node /ganache_db /ganache_db
+
+EXPOSE 8545/tcp
+
+ENTRYPOINT ganache-cli -k istanbul -l 0x7A1200 -h 0.0.0.0 -a 310 -i 54321 --db $GANACHE_DB_PATH
```

Comment:
@adamsoffer mentioned that deterministic account generation (i.e. with the `-d` flag) would be helpful for his subgraph integration test suite.

Reply:
the contract addresses will only change if you rebuild the image but I can see the benefit of _always_ having the same contract addresses.  Fixed in 25e0ca1